src/uti.ts

```diff
@@ -115,7 +115,18 @@ export const formatTable = (
 
   return `${tableHeader}\n${tableSeparator}\n${gameRows}\n`;
 };
-
+export const formatStatsTable = (stats: Stats): string => {
```

Comment:
I see we have `best` and `last` for the rating.
What do you think about showing both in the table? Maybe add a column **Type** or something similar and two rows, one for **Best** and one for **Last**.